gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java

```diff
@@ -39,7 +40,20 @@
     private List<UserDetailRole> roles;
     private String username;
     private byte[] picture;
+    /**
+     * The user creation date
+     */
+    private Date createdAt;
```

Comment:
Please force the property name to `created_at` for output as we are doing for other entities (see https://github.com/gravitee-io/gravitee-management-rest-api/blob/master/gravitee-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/ApiEntity.java#L102).